src/components/HorizontalTab/index.tsx

```diff
@@ -17,25 +17,25 @@ const HorizontalTab = ({selectedTab, onSelectedTabChange, tabs = []}: Horizontal
   const styles = StyleSheet.create({
     tabs: {
       width,
-      borderBottomColor: COLORS.lightgrey,
+      borderBottomColor: COLORS.gray100,
       borderBottomWidth: 1,
       flexDirection: 'row',
       alignItems: 'center',
       justifyContent: 'center',
-      backgroundColor: COLORS.white,
+      backgroundColor: COLORS.almostBlack,
       display: 'flex'
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color changes from `COLORS.lightgrey` to `COLORS.gray100` and from `COLORS.white` to `COLORS.almostBlack` seem appropriate for the styling of the tabs. However, please ensure that these color changes are consistent with the overall theme of the application and provide a good user experience in terms of readability and contrast.

```diff
-      borderBottomColor: COLORS.lightgrey,
+      borderBottomColor: COLORS.gray100,
       borderBottomWidth: 1,
       flexDirection: 'row',
       alignItems: 'center',
       justifyContent: 'center',
-      backgroundColor: COLORS.white,
+      backgroundColor: COLORS.almostBlack,
       display: 'flex'
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->